routers/web/web.go

```diff
@@ -403,6 +403,14 @@ func registerRoutes(m *web.Route) {
 		}
 	}
 
+	userSettingModuleEnabled := func(module string) func(ctx *context.Context) {
```

Comment:
It seems an over-engendering. It could be simply checked in `user_setting.DeleteAccount`, instead of bloating the "web.go", it has been much too large.